From 7ae1e3e90b5424a334c1b73ecf28a6b231f4ff6e Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Thu, 13 Oct 2016 18:31:26 +0800
Subject: [PATCH 1207/5242] MLK-13339-2 ARM: dts: pxp: enable pxp on imx6sll
 lpddr3 arm2 board

commit  43b864388e1f9c86a8846d1bff1122a06f2044a9 from
https://source.codeaurora.org/external/imx/linux-imx.git

correct the clock name for pxp and enable pxp

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts |    4 ++++
 arch/arm/boot/dts/imx6sll.dtsi            |    2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts b/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts
index e18de06..8e3fcc5 100644
--- a/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts
@@ -451,6 +451,10 @@
 	};
 };
 
+&pxp {
+	status = "okay";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1>;
diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index 8da2b38..e7ce8dc 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -616,7 +616,7 @@
 					<GIC_SPI 102 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SLL_CLK_DUMMY>,
 					 <&clks IMX6SLL_CLK_PXP>;
-				clock-names = "pxp-ipg", "disp-axi";
+				clock-names = "pxp_ipg", "pxp_axi";
 				status = "disabled";
 			};
 
-- 
1.7.9.5

